x-pack/plugins/enterprise_search/server/lib/enterprise_search_config.test.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockReadFileSync } from '../__mocks__/fs.mock';
+
+import http from 'http';
+import https from 'https';
+
+import { EnterpriseSearchConfig, config } from './enterprise_search_config';
+
+// -----------------------------------------------------------------------------
+test('set correct defaults', () => {
+  const configValue = new EnterpriseSearchConfig(config.schema.validate({}));
+  expect(configValue).toMatchSnapshot({
```

Comment:
For better or for worse, we're not really using Jest snapshots in our codebase currently and I don't super think we need it here just for a basic object check. You should be able to change this to

```suggestion
  expect(configValue).toEqual({
```

+ delete the generated snapshot, and it should continue to work/pass as expected.